packages/flutter/lib/src/widgets/page_storage.dart

```diff
@@ -129,7 +113,123 @@ class PageStorageBucket {
   }
 }
 
-/// A widget that establishes a page storage bucket for this widget subtree.
+/// Establish a subtree in which widgets can opt into persisting states after
+/// being destroyed.
+///
+/// [PageStorage] is used to save and restore values that can outlive the widget.
+/// For example, when multiple pages are grouped in tabs, when a page is
+/// switched out, its widget is destroyed and its state is lost. By adding a
+/// [PageStorage] at the root and adding a [PageStorageKey] to each page, few
```

Comment:
how about: "...some of the page's state (e.g. the scroll position of a [Scrollable] widget) will be stored automatically...

Reply:
This comment is still unaddressed.